rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml

```diff
@@ -11,7 +11,7 @@ Exploitation involves chaining multiple primitives to load an arbitrary DLL into
 SYSTEM.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
The integration is only parsing DWORD and QWORD registry values, so registry.data.strings will always be null here. Dropping Support